packages/web_drivers/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ environment:
 
 dev_dependencies:
```

Comment:
I haven't noticed this before, but I think this should be `dependencies`. `dev_dependencies` are only those used for testing.

Reply:
ah, I thought dev_dependencies for all dev related stuff. Thanks I will change it.

Reply:
done.